.github/workflows/continuous-integration.yml

```
@@ -73,6 +69,64 @@ jobs:
           name: "phpunit-sqlite-${{ matrix.php-version }}-${{ matrix.dbal-version }}-coverage"
           path: "coverage*.xml"
 
+  phpunit-oci8:
+    name: "PHPUnit on OCI8"
+    runs-on: "ubuntu-20.04"
+    #needs: "phpunit-smoke-check"
+
+    strategy:
+      matrix:
+        php-version:
+          - "7.4"
+        oracle-version:
+          - "11"
+        dbal-version:
+          - "2.12"
```

We don't support DBAL 2.12 anymore. Let's try to make the tests work with DBAL 3 first and if we succeed, we can consider fixing the build with DBAL 2.13. But I'm not really sure if that's worth the effort tbh.

```suggestion
          - "default"
```

---

> We don't support DBAL 2.12 anymore. Let's try to make the tests work with DBAL 3 first and if we succeed, we can consider fixing the build for DBAL 2.13. But I'm not really sure if that's worth the effort tbh.

I'm basically trying to run tests without errors. There are strange bugs due to Oracle features. For example: Doesn't delete entries in bulk, because it waits for an explicit commit until the session closes, which is why the test fails.

Lots of work here

---

Right, but it doesn't get better if you force the CI to run on abandoned versions of DBAL only. Please apply my change and have the CI run again.